pkg/cli/admin/prune/deployments/data.go

```diff
@@ -90,7 +96,7 @@ func FilterZeroReplicaSize(item metav1.Object) bool {
 	case *kappsv1.ReplicaSet:
 		return *v.Spec.Replicas == 0 && v.Status.Replicas == 0
 	default:
-		return true
+		return false
```

Comment:
Honestly both `true` and `false` will be fine here, since a) theoretically we shouldn't get a different resource than RS or RC here and b) even if we did it will be filtered out elsewhere. Still it would be nice to see a comment here why one was picked over the other.